acceptance/openstack/autoscaling/v2/policies_test.go

```diff
@@ -23,10 +23,11 @@ func TestPolicyLifecycle(t *testing.T) {
 	if networkID == "" || vpcID == "" {
```

Comment:
=== RUN   TestPolicyLifecycle
    helpers.go:31: Attempting to create AutoScaling Group
    helpers.go:34: Created AutoScaling Group: d6e0153f-08a8-4d70-a996-3ccddf0cb9e2
    policies_test.go:49: Attempting to create AutoScaling Policy
    policies_test.go:52: Created AutoScaling Policy: 3c647951-d322-4e2a-b601-70f1aba3b4f0
    policies_test.go:67: Attempting to update AutoScaling policy
    policies_test.go:84: Updated AutoScaling Policy
    tools.go:72: {
          "scaling_policy_id": "3c647951-d322-4e2a-b601-70f1aba3b4f0",
          "scaling_policy_name": "as-policy-update-9PW",
          "scaling_resource_id": "d6e0153f-08a8-4d70-a996-3ccddf0cb9e2",
          "scaling_resource_type": "SCALING_GROUP",

          "policy_status": "INSERVICE",
          "scaling_policy_type": "RECURRENCE",
          "alarm_id": "",
          "scheduled_policy": {
            "launch_time": "10:30",
            "recurrence_type": "Weekly",
            "recurrence_value": "1,3,5",
            "start_time": "",
            "end_time": "2040-12-31T10:30Z"
          },
          "scaling_policy_action": {
            "operation": "ADD",
            "size": 0,
            "percentage": 30,
            "limits": 0
          },
          "cool_down_time": 300,
          "create_time": "2022-11-04T14:37:16Z",
          "meta_data": {
            "metadata_bandwidth_share_type": "",
            "metadata_eip_id": "",
            "metadata_eip_address": ""
          }
        }
    policies_test.go:55: Attempting to delete AutoScaling Policy
    policies_test.go:58: Deleted AutoScaling Policy: 3c647951-d322-4e2a-b601-70f1aba3b4f0
    helpers.go:40: Attempting to delete AutoScaling Group
    helpers.go:45: Deleted AutoScaling Group: d6e0153f-08a8-4d70-a996-3ccddf0cb9e2
--- PASS: TestPolicyLifecycle (51.50s)

Reply:
passed